Repairs/app/src/main/java/com/rooio/repairs/Equipment.kt

```diff
@@ -17,23 +32,69 @@ class Equipment : NavigationBar() {
                 ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT))
 
         //sets the action bar onto the page
-
         val actionbar_inflater = layoutInflater
         val poopView = actionbar_inflater.inflate(R.layout.action_bar, null)
         window.addContentView(poopView,
                 ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT))
 
         supportActionBar!!.elevation = 0.0f
 
-
+        //making navigation bar w/ Equipment text highlighted
         createNavigationBar("equipment")
+
+        equipmentListView = findViewById<View>(R.id.equipmentList) as ListView
+
+
+        //get equipment
+        loadEquipment();
+    }
+
+    private fun loadEquipment() {
+        //val url = "https://capstone.api.roopairs.com/v0/service-locations/" + getUserLocationID().toString() + "/equipment/"
+        val url = "https://capstone.api.roopairs.com/v0/service-locations/QPKrjKN/equipment/"
+
+        var responseFunc = Function<Any, Void?> { jsonResponse: Any? ->
+            try {
+                val jsonArray = jsonResponse as JSONArray
+                loadElements(jsonArray)
+            } catch (e: JSONException) {
+                e.printStackTrace()
+            }
+            null
+        }
+
+        var errorFunc = Function<String, Void?> { string: String? ->
+            null
+        }
+
+        val request = JsonRequest(false, url, null, responseFunc, errorFunc, true)
+        requestGetJsonArray(request)
+    }
+
+
+    private fun loadElements(response: JSONArray) {
+        equipmentList.clear()
+        for (i in 0 until response.length()) {
+            val equipment = EquipmentData(response.getJSONObject(i))
+            equipmentList.add(equipment);
+        }
+
+        val customAdapter = EquipmentCustomAdapter(this, equipmentList)
+        if (equipmentList?.size != 0) equipmentListView!!.adapter = customAdapter
     }
 
     override fun animateActivity(boolean: Boolean){
```

Comment:
I do not think animations work with constraint layouts, maybe consider adjusting the width of the box? The XD does not show what it looks like with the nav bar open, so I would just implement something similar to what was done for the dashboard